controllers/provider-gcp/example/Infrastructure.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: shoot--gcp--foo
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: shoot--gcp--foo
+  name: core-gcp
+type: Opaque
+data:
+# serviceaccount.json: base64(serviceaccount-json)
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Cluster
+metadata:
+  namespace: shoot--gcp--foo
+  name: shoot--gcp--foo
+spec:
+  cloudProfile:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: CloudProfile
+  seed:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: Seed
+  shoot:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: Shoot
+    spec:
+      cloud:
+        gcp:
+          networks:
+            pods: 10.243.128.0/17
+            services: 10.243.0.0/17
+    status:
+      lastOperation:
+        state: Succeeded
+
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Infrastructure
+metadata:
+  namespace: shoot--gcp--foo
+  name: gcp-infra
+spec:
+  type: gcp
+  region: europe-west1
+  secretRef:
+    namespace: shoot--gcp--foo
+    name: core-gcp
+  providerConfig:
+    apiVersion: gcp.provider.extensions.gardener.cloud/v1alpha1
+    kind: InfrastructureConfig
+    networks:
+      worker: 10.242.0.0/19
```

Comment:
Similar to https://github.com/gardener/gardener/blob/master/example/90-shoot-gcp.yaml#L17 we should have `internal` here as well

Reply:
`internal` is not used, so why have it? In the `aws` example, it's also not a fully functioning shoot.